distro/superbuild/cmake/externals.cmake

```diff
@@ -238,9 +265,18 @@ ExternalProject_Add(PythonQt
 
 ###############################################################################
 # ctkPythonConsole
+set(ctkPythonConsole_REPO)
+set(ctkPythonConsole_TAG)
+if(${DD_QT_VERSION} VERSION_GREATER "4")
+  set(ctkPythonConsole_REPO https://github.com/mwoehlke-kitware/ctkPythonConsole)
+  set(ctkPythonConsole_TAG qt5)
```

Comment:
On closer inspection, it seems this would need tweaking to work with either Qt4 or Qt5. I don't know that I'll be able to do that; @sankhesh, can you do it?